test/sequential/test-fs-largefile.js

```diff
@@ -34,7 +34,7 @@ const fd = fs.openSync(filepath, 'w+');
 const offset = 5 * 1024 * 1024 * 1024; // 5GB
 const message = 'Large File';
 
-fs.truncateSync(fd, offset);
+fs.ftruncateSync(fd, offset);
```

Comment:
Is this change intended?

Reply:
Yes. `fs.truncateSync()` emits a deprecation warning when called with a file descriptor. I'll move it to a separate commit. Probably should have been two commits to begin with.